packs/data/pfs-season-1-bestiary.db/unkillable-zombie-brute-pfs-1-01.json

```diff
@@ -396,46 +396,6 @@
             "sort": 600000,
             "type": "action"
         },
-        {
-            "_id": "V5U9d0DWWw5xvPxJ",
-            "data": {
-                "actionCategory": {
-                    "value": "defensive"
-                },
-                "actionType": {
-                    "value": "passive"
-                },
-                "actions": {
-                    "value": null
-                },
-                "description": {
-                    "value": "<p>This zombie is nigh unkillable.</p>\n<p>The zombie loses its weakness to slashing and gains resistance 3 against all damage, and it gains weakness 6 to critical hits.</p>"
-                },
-                "requirements": {
-                    "value": ""
-                },
-                "rules": [],
-                "slug": null,
-                "source": {
-                    "value": ""
-                },
-                "traits": {
-                    "custom": "",
-                    "rarity": "common",
-                    "value": []
-                },
-                "trigger": {
-                    "value": ""
-                },
-                "weapon": {
-                    "value": ""
-                }
-            },
-            "img": "systems/pf2e/icons/actions/Passive.webp",
-            "name": "Unkillable",
```

Comment:
This shouldn't be removed, rather it should be replaced from the Bestiary Family Ability Glossary